_maps/map_files/CTF/limbo.dmm

```diff
@@ -894,7 +894,7 @@
 /obj/effect/turf_decal/siding/yellow,
 /obj/structure/showcase{
 	desc = "A historical figure of great importance to the wizard federation. He spent his long life learning magic, stealing artifacts, and harassing idiots with swords. May he rest forever, Rodney.";
-	icon = 'icons/mob/mob.dmi';
+	icon = 'icons/mob/simplemob/mob.dmi';
 	icon_state = "nim";
 	name = "wizard of yendor showcase"
 	},
```

Comment:
We have the fucking icon var_edited? Fuck!

Do you think you'd be willing to create a subtype of /obj/structure/showcase with these exact variables applied? This same thing shows up in a map key somewhere else in the map, as well as in CentCom.dmm. Then, replace these instances completely with the new subtype? Let me know if you need help figuring out how to do this, it's a great step to ensuring that if we want to change around icons again, we don't have to touch maps (we also shouldn't have done this in the first place).

There's also the statue that you change in IceBoxStation.dmm. That's also a good candidate for the subtype.

Reply:
Done -- I'll need a review of that stuff, since I don't really know what I'm doing when it comes to map edits.